nuxeo-coldstorage-web/pom.xml

```diff
@@ -10,68 +10,67 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
+        <version>1.15.1</version>
```

Comment:
Should we consider defining the plugin version in the nuxeo parent pom?

Reply:
which was removed by https://hyland.atlassian.net/browse/NXP-31492 -> https://github.com/nuxeo/nuxeo-lts/commit/a7694c90303f905a1e3a2bcb45cd4ff854e1c277

Reply:
If we can support not having it in the nuxeo-ecm root pom it would be great.

Reply:
> Furthermore, it could be interesting to have a Maven property for `nodeVersion`.
Definitely, and where would we define this Maven property, in the nuxeo-parent pom?